README.md

```diff
@@ -7,14 +7,27 @@ Gearbox.js
 [![Build Status](https://travis-ci.org/cognitedata/gearbox.js.svg?branch=master)](https://travis-ci.org/cognitedata/gearbox.js)
 [![codecov](https://codecov.io/gh/cognitedata/gearbox.js/branch/master/graph/badge.svg)](https://codecov.io/gh/cognitedata/gearbox.js)
 
-## Setup
+## Install package
 
-Run `yarn`.
+Run `yarn add @cognite/gearbox` or `npm install @cognite/gearbox --save`.
 
 Install additional dependencies:
-1.  AntD by running `yarn add antd` or `npm i antd --save`.
-2.  Styled-components via `yarn add styled-components` or `npm i styled-components --save`.
+1.  Cognite SDK (JS) `yarn add @cognite/sdk` or `npm install @cognite/sdk --save`
+2.  AntD by running `yarn add antd` or `npm i antd --save`.
+3.  Styled-components via `yarn add styled-components` or `npm i styled-components --save`.
+
+## Setup
+
+Run `yarn`
```

Comment:
We can remove this part. No-one using the package need to run this (only contributors as ourself) and we know this.

Reply:
Removed